stdlib/Profile/docs/src/index.md

```diff
@@ -20,6 +20,14 @@ Profile.clear_malloc_data
 
 ## Memory profiling
 
+Note: The current implementation of the Allocations Profiler _does not
+capture types for all allocations._ Allocations for which the profiler
+could not capture the type are represented as having type
+`Profile.Allocs.UnknownType`.
+
+You can read more about the missing types and the plan to improve this, here:
+https://github.com/JuliaLang/julia/issues/43688.
+
```

Comment:
Maybe put this into the docstring for `Profile.Allocs.@profile` instead? That way people will see it in both spots, and also that way it can come _after_ the main explanation of how this works, just as like a little note :)

Reply:
done